.github/copilot-instructions.md

```diff
@@ -66,6 +138,27 @@ src/
 - Small, focused components.
 - Separate all logic from JSX.
 - Include proper loading and error states.
+- **Component props structure**:
+  - Props should be an object with type `Props`
+  - `Props` type should be defined and exported within the component file
+  - Example:
+    ```tsx
+    export type Props = {
+      title: string
+      isVisible: boolean
+      onClose: () => void
+    }
+
+    export default function Modal(props: Props) {
+      const { title, isVisible, onClose } = props
+      // Component implementation
+    }
+    ```
```

Comment:
```suggestion
- **Component props structure**:
  - Props type should be defined and exported within the component file
  - Props should be destructured within the component parameter list.
```

We're repeating the part about the type of props. Instead lets use this section to define how its used and exported and leave the typescript part to the above section.